types/critters-webpack-plugin/index.d.ts

```diff
@@ -1,8 +1,13 @@
-import { Plugin } from "webpack";
+import { Compiler, WebpackPluginInstance } from "webpack";
 
 export default Critters;
-declare class Critters extends Plugin {
+declare class Critters implements WebpackPluginInstance {
     constructor(options?: Critters.CrittersOptions);
+    apply: (compiler: Compiler) => void;
+    getCssAsset(href: any, style: any): Promise<any>;
```

Comment:
Based on [definition](https://github.com/GoogleChromeLabs/critters/blob/05f5a48d8bcec091cda37fa4fc97a895b33db833/packages/critters-webpack-plugin/src/index.js#L146) first argument is an string and second one an HTMLStyleElement